### PR TITLE
Add AARCH64 support in FFI

### PR DESCRIPTION
**Pull request by yuqilinaro**
This PR is to add support for the AARCH64/ARM64 in JRuby FFI.

### CODE REVIEW

**Comment by yuqilinaro** [comment]
@ayappanec  Is this patch ok to be merged for support for AArch64?
Could you please review it? thanks a lot!

**Comment by ayappanec** [comment]
The PR looks good.

**Comment by yuqilinaro** [comment]
@headius Could you please check it? Thanks!

File: core/src/main/java/org/jruby/ext/ffi/Platform.java [hunk 1]
```diff
@@ -84,6 +84,7 @@
         SPARCV9,
         S390X,
         ARM,
+	AARCH64,
```
Comment (olleolleolle) [inline core/src/main/java/org/jruby/ext/ffi/Platform.java]: This is a tab, use spaces instead.

Comment (ayappanec) [inline core/src/main/java/org/jruby/ext/ffi/Platform.java]: Interesting !. How did you figured this out ? by seeing the indentation ?

Comment (olleolleolle) [inline core/src/main/java/org/jruby/ext/ffi/Platform.java]: The presentation was 1 character off in _this_ view.

Then: Visiting "View", marking and dragging the cursor over and seeing that the visualization of the marked text "snapped" to a tab stop, not to individual spaces.

Comment (ayappanec) [inline core/src/main/java/org/jruby/ext/ffi/Platform.java]: This is cool. Thanks for the info.

Comment (yuqilinaro) [inline core/src/main/java/org/jruby/ext/ffi/Platform.java]: @olleolleolle  It's  awesome! Thanks!

**Comment by guyuqi** [comment]
@enebo Could you please take your time to review my changes? Thanks!

**Comment by headius** [comment]
Hello! Sorry this slipped a bit. Looking into merging.

**Pull request merged**